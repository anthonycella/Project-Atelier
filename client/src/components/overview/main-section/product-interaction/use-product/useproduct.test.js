import renderer from 'react-test-renderer';
import UseProduct from './useproduct.jsx';

it('Use Product component renders to the page', () => {
  const styleSelected = { name: 'Tyrone' };
  const stock = {
    '12345' : {
      size: 'M',
      quantity: 67,
    },
    '11245' : {
      size: 'M',
      quantity: 0,
    },
    '90345' : {
      size: 'M',
      quantity: 78,
    },
    '145645' : {
      size: 'M',
      quantity: 12,
    },
    '12345' : {
      size: 'M',
      quantity: 90,
    }
  };

  const component = renderer.create(
    <UseProduct stock={stock} styleSelected={styleSelected} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
