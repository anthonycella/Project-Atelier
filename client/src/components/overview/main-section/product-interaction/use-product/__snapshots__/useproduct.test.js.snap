// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Use Product component renders to the page 1`] = `
<div
  className="sc-gKXOVf gaiTkr"
>
  <div
    className="sc-dkzDqf jNClnl"
  >
    <select
      className="sc-bczRLJ bvsNxk"
      defaultValue="Select Size"
      onChange={[Function]}
    >
      <option
        hidden={true}
        selected={true}
        value={0}
      >
        Select Size
      </option>
      <option
        value="12345"
      >
        M
      </option>
      <option
        value="90345"
      >
        M
      </option>
      <option
        value="145645"
      >
        M
      </option>
    </select>
    <div
      className="sc-jSMfEi sc-iBkjds chQbUE ewQnNH"
    >
      -
    </div>
  </div>
  <div
    className="sc-eCYdqJ cBRphG"
  >
    <div
      className="sc-hKMtZM dwHzza"
      onClick={[Function]}
    >
      ADD TO CART
    </div>
  </div>
</div>
`;
